src/TMediaData.h

```diff
@@ -84,6 +92,8 @@ class TMediaData
     QString mMediaTag;
     QString mMediaUrl;
     QString mMediaKey;
+    int mMediaFadeIn;
+    int mMediaFadeOut;
```

Comment:
Apropos my prior comment - shouldn't the declaration be these instead (also setting the default constructed values - which is better than doing it in the *body* of the `TMediaData` constructor!):
```suggestion
    enum MediaFade mMediaFadeIn = MediaFadeNotSet;
    enum MediaFade mMediaFadeOut = MediaFadeNotSet;
```
?

Reply:
Made this change, thank you.